core/src/components/searchbar/searchbar.tsx

```diff
@@ -100,7 +100,7 @@ export class Searchbar implements ComponentInterface {
    * Set the cancel button icon. Only available when the theme is `"md"`.
    * Defaults to `"arrow-back-sharp"`.
    */
-  @Prop() cancelButtonIcon = config.get('backButtonIcon', arrowBackSharp) as string;
```

Comment:
I think we should use `backButtonIcon` as a fallback if the config for `searchbarCancelIcon` is not set because this is useful for Material Design to keep them in sync: [Search specs](https://m3.material.io/components/search/specs#91980222-3d9b-4d8c-9900-df0a498c150d)

Reply:
https://github.com/ionic-team/ionic-framework/pull/29373/commits/e2a8046a33c734d24010a655b2701b1e06a71d1c